src/apps/chifra/pkg/pinlib/manifest/manifest.go

```diff
@@ -0,0 +1,36 @@
+package manifest
+
+import (
+	"os"
+
+	"github.com/TrueBlocks/trueblocks-core/src/apps/chifra/pkg/config"
+)
+
+type IpfsHash = string
```

Comment:
Not sure either. Maybe they need `[]byte` more often, because of sending data to RPC and decoding/encoding hexes and addresses, so they don't want to type cast?